packages/settings-library/src/settings_library/redis.py

```diff
@@ -42,6 +42,6 @@ def build_redis_dsn(self, db_index: RedisDatabase) -> str:
                 ),
                 host=self.REDIS_HOST,
                 port=self.REDIS_PORT,
-                path=f"/{db_index}",
+                path=f"{db_index}",
```

Comment:
is this a deja-vu ? I saw this ian a different pR :-)

Reply:
yes. I took the opportunity to fast-track some fixes